modules/core/src/main/scala/org/scalasteward/core/vcs/data/Repo.scala

```diff
@@ -17,23 +17,30 @@
 package org.scalasteward.core.vcs.data
 
 import io.circe.{KeyDecoder, KeyEncoder}
+import org.scalasteward.core.git.Branch
 
 final case class Repo(
     owner: String,
-    repo: String
+    repo: String,
+    branch: Option[Branch]
```

Comment:
```suggestion
    branch: Option[Branch] = None
```
This would avoid many changes in the tests.